src/components/shifts/ShiftForm.vue

```diff
@@ -71,12 +71,34 @@
           :error="endError"
         />
       </v-col>
+      <v-col cols="12" class="ma-0">
+        <v-expand-transition hide-on-leave>
+          <ClockCardAlert
+            v-if="messages.length !== 0"
+            :messages="messages"
+            :type="alertType"
+          ></ClockCardAlert>
+        </v-expand-transition>
+        <v-expand-transition>
+          <v-row v-if="!enoughBreaktime" align="center">
```

Comment:
Wäre das ein Kandidat für eine eigene View? Oder lohnt sich das hier nicht?
Das ShiftForm ist ja auch sonst aus recht kleinteiligen Stücken aufgebaut.

Reply:
Evtl. ja, ich bin nicht so fit mit der Upward Parameter propagation.

Reply:
Du meinst Komponente, oder?

Reply:
Ja, also so ein `<ShiftBreakWarning>` oder so. Parameter Propagation upward ist etwas unintuitiv. Ist aber auch nicht so wichtig, wäre granularer, aber macht auch nicht so viel übersichtlicher.

Reply:
Der Checkbox wert muss halt in der Form sein, aber das wollte ich sowieso mal ausprobieren.